membership-common/src/main/scala/com/gu/memsub/subsv2/services/SubscriptionService.scala

```diff
@@ -54,74 +64,210 @@ object Trace {
 
 }
 
-import com.gu.memsub.subsv2.services.Trace.Traceable
+class SubscriptionService[M[_]: Monad](pids: ProductIds, futureCatalog: => M[CatalogMap], rest: SimpleClient[M], soap: SoapClient[M])
```

Comment:
all this was deleted as unused last month, but it turned out it was diplicated in the main codebase, so this puts it back again